feature-group-definitions/html-media-capture.yml

```
@@ -1,4 +1,5 @@
 name: HTML media capture
+description: The `capture` HTML attribute for `<input type="file">` elements allows the user to, optionally, capture media using the device's camera, microphone, or other media-capture capabilities.
```

Trying to simplify a bit to the concrete:

```suggestion
description: The `capture` HTML attribute for `<input type="file">` elements allows the user to capture media using the device's camera or microphone.
```

If there's something beyond camera or microphone implemented in some browser, we can name that directly.